doctum.conf.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+// Prevent e.g. 'Notice: Constant MATH_BIGINTEGER_MONTGOMERY already defined'.
+class MyArrayStore extends Doctum\Store\ArrayStore
+{
+    public function removeClass(Doctum\Project $project, $name)
+    {
+        unset($this->classes[$name]);
+    }
+}
+
+$root = realpath(__DIR__) . '/';
+
+$iterator = Symfony\Component\Finder\Finder::create()
+    ->files()
+    ->name('*.php')
+    ->in($root . 'phpseclib');
+
+$versions = Doctum\Version\GitVersionCollection::create($root)
+    ->add('1.0')
+    ->add('2.0')
+    ->add('master');
```

Comment:
Added to fix https://github.com/phpseclib/phpseclib/blob/master/README.md#documentation